src/platform/plugins/shared/data/public/search/session/sessions_mgmt/components/table/columns/name.tsx

```diff
@@ -125,9 +120,11 @@ export const nameColumn = ({
           }}
         >
           <TableText data-test-subj="sessionManagementNameCol">
-            {name}
-            {notRestorableWarning}
-            {versionIncompatibleWarning}
+            <EuiFlexGroup gutterSize="s">
```

Comment:
```suggestion
            <EuiFlexGroup gutterSize="s" responsive={false} alignItems="center">
```
Have not tested yet but might be better this way.

Reply:
tried it now but `align-items` doesn't do anything for the icon so it looks the same

Reply:
that works, updated 😄